### ISSUE TITLE
test dataset

### ISSUE THREAD

**Issue by wwjwy** [lorenmt/reco#16]
Hi,
In your voc dataset split, I noticed that there are only training and val set. Are there additional test sets to verify model performance after you have completed training 200 epochs? The voc result in Table 1 in the paper is on the validation set or the test set? 
Thanks!

**Comment by wwjwy** [comment]
In addition, I print the confmatrixs, but i am confused which is correct as show below.
![DC93EAC6B62CEA10311DB5E67B63917E](https://user-images.githubusercontent.com/66406998/149093197-d0d77d0d-ac52-4f48-9003-cebd4771452d.png)
the left or the right is the correct confmatrixs?
"conf_mat.update(pred.argmax(1).flatten(), test_label.flatten())"

**Comment by lorenmt** [comment]
All results are reported from validation set averaged across three random seeds. This is the standard in semi-supervised segmentation. 

For your second question, it should be the second graph.. but it doesn't really matter as long as you keep consistent with the choice of the dimension.

**Comment by wwjwy** [comment]
Thanks a lot!
